web/Web/web/static/js/highlight.js

```diff
@@ -182,6 +182,59 @@ $(function() {
     marked_matches_counter = {};
   }
 
+  /** This function processes the entered keyword to support both
```

Comment:
Minor edit. Just to follow standard commenting style, please add new line after `/**`

Reply:
@XinyiYan Can you edit this? I'll merge this pull request right after.

Reply:
Done.